articles/how-to-optimize-wordpress-website-speed/index.md

```diff
@@ -0,0 +1,214 @@
+The ultimate measure of websites performance is loading speed. This is the amount
+of time it takes to load a web page. Web users are relying more on
+mobiles to access web content and services and thus more demand for optimized websites. If you run any website, this
```

Comment:
manage any website**s** would flow better. Sorry, I should have been clearer. This occurrence of website should be plural not the first one.